spec/observables/race-spec.js

```diff
@@ -0,0 +1,160 @@
+/* globals expect, it, describe, hot, cold, expectObservable */
+
+var Rx = require('../../dist/cjs/Rx');
+var Observable = Rx.Observable;
```

Comment:
Great looking tests. Can you also add the cases "should allow unsubscribing early" and "should not break unsubscription chain when unsubscribing"